### ISSUE TITLE
If no directory specified, give warning/error message

### ISSUE THREAD

**Issue by pzelnip** [tonybaloney/wily#18]
I did a:

```
wily build -n 2000
```
to produce metrics over the last 2000 commits.  That finished quickly, but of course then any report showed no data as I failed to specify the directory to analyze.

It took me a bit to figure that out (I assumed that since it coincided with using the -n param that I had stumbled across a bug). I think it'd be better if wily spat out a warning or an error if one leaves off the target(s) to analyze.  Something like:

```
$ wily build

WARNING: no target(s) specified, do you want to do "wily build somepath/"?

Found 150 revisions from 'git' archiver in '/path/to/your/repo'.
Running operators - raw,maintainability,cyclomatic
Processing |#                               | 10/450
```

**Comment by tonybaloney** [comment]
Fixed in 1.5.0.dev0

**Comment by tonybaloney** [comment]
released in 1.5.0
